packages/api/controllers/nomination.js

```diff
@@ -179,15 +179,14 @@ const syncNominations = async (req, res) => {
 const emailVerification = async (req, res) => {
   try {
     const { token } = req.params;
-    const { nomination: id } = jwt.verify(token, process.env.JWT_SECRET);
+    const { data: id } = jwt.verify(token, process.env.JWT_SECRET);
     await db.Nomination.update({ emailValidated: true }, { where: { id } });
-    return res.status(200).json({ status: 'ok' });
+    return res.status(200).end();
```

Comment:
For something like this I normally return a 204 instead of 200. Per the  HTTP spec:
```
The 204 (No Content) status code indicates that the server has
   successfully fulfilled the request and that there is no additional
   content to send in the response payload body.
```
https://datatracker.ietf.org/doc/html/rfc7231#section-6.3.5

Reply:
got it, will update test file and nomination file